### ISSUE TITLE
OpenSUSE: Undefined symbols in cython-hidapi

### ISSUE THREAD

**Issue by Ban3** [Ban3/python-evic#9]
@netman69 I'm opening this as a separate issue for better tracking.

Did you have the correct headers installed? On opensuse the packages you need for cython-hidapi are:  `libusb-1_0`, `libusb-1_0-devel` and `libudev-devel`.

**Comment by ReservedField** [comment]
Might be an issue with `/usr/lib` and `/usr/lib64`, as he's having with evic-sdk: ReservedField/evic-sdk#10

**Comment by netman69** [comment]
Heh, I got it sorted and fixed! And I did have the right packages, but also the wrong evil malicious ones: http://paste.dott.co.in/#dd4cedf52f58b900e8b6278073171ff4pwqZ7b7e
So there was a symlink to some strange libusb, that symlink is installed by mplab-x to their own messed up libusb (what where they thinking?!). So the steps i needed to install python-evic: install python3-devel, remove mplab-x, follow instructions :P.
